### PR TITLE
Implemented Google Tag Manager aka GTM trackers.

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

**Comment by tgalopin** [comment]
@hhamon is this ready?

**Comment by hhamon** [comment]
Normally yes!

**Comment by hhamon** [comment]
Just rebased the PR onto master.

File: app/Resources/views/donation/informations.html.twig [hunk 1]
```diff
@@ -113,3 +113,20 @@
         {{ form_end(form) }}
     </div>
 {% endblock %}
+
+{% block final_javascripts %}
+    {% if google_tag_manager_api_key is not empty %}
+        {{ include('third_party/gtm.html.twig') }}
+        <script type="text/javascript">
+            dataLayer.push({
+                'product':[{
```
Comment (mykiwi) [inline app/Resources/views/donation/informations.html.twig]: keys dont need to be inside quotes

File: app/Resources/views/third_party/gtm.html.twig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{# Google Tag Manager #}
+<script type="text/javascript">
+    var dataLayer = [];
+</script>
```
Comment (mykiwi) [inline app/Resources/views/third_party/gtm.html.twig]: you could keep 1 script tag.

File: src/AppBundle/Donation/DonationRequest.php [hunk 3]
```diff
@@ -6,10 +6,14 @@
 use libphonenumber\PhoneNumber;
 use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumber as AssertPhoneNumber;
```
Comment (mykiwi) [inline src/AppBundle/Donation/DonationRequest.php]: Could you sort them?

**Comment by hhamon** [comment]
@tgalopin any status on this one?

**Comment by hhamon** [comment]
@tgalopin shall we close this PR?

**Comment by marineb** [comment]
@hhamon @tgalopin hi! what happened with this one? We're closing it but it hasn't been merged, right? Or is there another PR that was closed?

**Comment by marineb** [comment]
@atamano should we close this one?

**Pull request closed without merging**